pages/masq/masq.md

```diff
@@ -0,0 +1,44 @@
+---
+name: "MASQ"
+logo: "masq".svg"
+color: "#003060"
+schemas:
+  - "masq"
+links:
+  - name: "Website"
+    link: "https://masq.ai/"
+  - name: "Medium"
+    link: "https://medium.com/masq-project"
+  - name: "Docs"
+    link: "https://docs.masq.ai/masq/"
+  - name: "Discord"
+    link: "https://discord.com/invite/masq"
+  - name: "Twitter"
+    link: "https://twitter.com/MASQ_ai"    
+tags:
+  - "MASQ DVPN"
+  - "Decentralized"
+dashboards:
+  - "dbustos20/masq-analytics-dashboard"
```

Comment:
doesn't work

Reply:
hey @philippWassibauer, we migrated the dashboard and forgot to update this. 

I changed it to `masq_network/masq-network-dashboard`